test/iron-dropdown.html

```diff
@@ -512,6 +526,50 @@
           });
         });
       });
+
+      suite('auto-cancel on blur', function() {
+
+        test('auto-cancel when focusable dropdown blurs', function(done) {
+          var elements = fixture('FocusableDropdownWithOtherFocusableElements');
+          var dropdown = Polymer.dom(elements).querySelector('iron-dropdown');
+          var content = Polymer.dom(dropdown).querySelector('.dropdown-content');
+          var button1 = Polymer.dom(elements).querySelector('#focusableButton1');
+          var button2 = Polymer.dom(elements).querySelector('#focusableButton2');
+
+          // Step 1: First button gets focus.
+          button1.addEventListener('focus', function() {
+            dropdown.open();
+            runAfterOpen(dropdown, function() {
+              // Opening the dropdown moves focus to the dropdown content, but
+              // not if the window doesn't have focus, so to cover that case we
+              // have to simulate a focus event.
+              MockInteractions.focus(content);
+            });
+          });
+
+          // Step 2: Dropdown content gets focus after opening.
+          content.addEventListener('focus', function() {
+            // Move focus outside the dropdown to trigger auto-cancel behavior.
+            // To cover both interactive and automated testing, we need to do
+            // both a real focus and simulated focus change.
+            button2.focus();
+            MockInteractions.focus(button2);
+          });
+
+          // Step 3: Dropdown closes after second button gets focus.
+          dropdown.addEventListener('iron-overlay-closed', function(event) {
+            // The iron-overlay-closed event fires before the dropdown closes,
+            // which is unfortunate. Wait before checking dropdown was closed.
+            Polymer.Base.async(function() {
```

Comment:
`iron-overlay-closed` should be fired after the dropdown is completely closed, if that's not the case it is a bug.
In which browser did you see this?